js/main.js

```diff
@@ -19,6 +19,86 @@ function submission(event) {
   formObject.entryId = data.nextEntryId;
   data.nextEntryId++;
   data.entries.unshift(formObject);
-  $image.src = 'images/placeholder-image-square.jpg';
+  $newImage.src = 'images/placeholder-image-square.jpg';
   $form.reset();
+  renderEntry(data.entries[0]);
+  toggleNoEntries();
+  viewEntries();
+}
+
+function renderEntry(entry) {
+  var $ul = document.querySelector('ul');
```

Comment:
You should not be appending to the `ul` in this function. You should be `return`ing the `li` element, and appending it to the `ul` after saving the return value when you call the function.